src/screens/SignInScreen.test.tsx

```diff
@@ -0,0 +1,56 @@
+import { SignInScreen } from './SignInScreen'
+
+import { act, cleanup, fireEvent, render } from '~utils/testUtils'
+
+afterEach(cleanup)
+
+describe('SignInScreen', () => {
+  it('should render SignIn screen with the key elements', () => {
+    const { getByPlaceholderText, getByRole, getAllByRole } = render(<SignInScreen />)
+    const emailInput = getByPlaceholderText('john@doe.com')
```

Comment:
I don't think that getting elements by placeholder text is good.

We should rather use `accessibilityRole`, '`accessibilitylabel`, `testID`

Reply:
What about this one?